### ISSUE TITLE
Internal Accessibility Audit

### ISSUE THREAD

**Issue by candilemoine** [department-of-veterans-affairs/va.gov-team#114786]
### Description
We had a request from the Platform team to submit a VA 508 office audit. We will complete a [full accessibility review](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/prepare-for-an-accessibility-staging-review) for ARP. 

### Blocked by 
- Homepage updates
- Login flow
- Help page updates

### Acceptance Criteria/Conditions of Satisfaction

- [ ] Verify test user account to use 
- [ ] Fill out the audit document and log all issues in there
- [ ] Audit using NVDA/Chrome
- [ ] Audit using Voiceover/Safari on Desktop
- [ ] Audit using Android Talkback on chrome
- [ ] Audit using either iPad or iPhone with Voiceover/Safari mobile
- [ ] List accessibility updates needed

----

Platform guidance: https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/prepare-for-an-accessibility-staging-review

### Who completed the use of color and color contrast test? 
[Tester] tested with the Chrome browser for both desktop and mobile (Android).
[Tester] tested with the Safari browser for both desktop (Macbook Air) and mobile (iPhone 13 Pro).

### Use of color and color contrast checks
Tools used: Wave and Axe extensions for Chrome, Colour Contrast Analyzer for Mac.

- [ ] All text of 20px or smaller has a 4.5:1 contrast ratio to its background (or better)
- [ ] All text of 20px or larger has a 3:1 contrast ratio to its background (or better)
- [ ] Non-text elements have a 3:1 contrast ratio to their background and to neighboring elements (or better)
- [ ] Color is not the only way to distinguish links from other text (eg. links are underlined)
- [ ] Any charts, maps, infographics, and tables convey all information without only relying on color
- [ ] Content does not refer to color, especially when providing user instructions (eg. "Click the blue button")

### How did color testing go?
Chrome:

Safari:

### Who completed the axe scans?
[Tester]

Axe Checks:

- [ ] Each page has been scanned using axe (results shared below)
- [ ] axe is integrated into your end-to-end testing

### Who completed the content zoom and reflow test?
[Tester] tested with the Chrome browser for both desktop and mobile (Android).
[Tester] tested with the Safari browser for both desktop (Macbook Air) and mobile (iPhone 13 Pro).

### Content zoom and reflow checks (desktop AND mobile)
**Desktop:**

- [ ] All page elements are readable and usable at 200% zoom
- [ ] All page elements are readable and usable at 300% zoom
- [ ] All page elements are readable and usable at 400% zoom

Successfully tested on desktop (Chrome and Safari) and all page elements are readable and usable at 200, 300, and 400% zoom.

**Mobile:**

- [ ] All page elements are readable and usable at 200% zoom
- [ ] All page elements are readable and usable at 300% zoom
- [ ] All page elements are readable and usable at 400% zoom Chrome on Android and Safari on iPhone both have a limit of 300%.

For both mobile browsers, many page elements are not readable and usable at 200% and 300% zoom. This includes:

- Issues with text not wrapping / getting cut off
- Issues with resizing
- Issues with content taking up the entire screen (Safari on iPhone at 300%)

More details, including screenshots, in the comments below.

### Who completed the keyboard navigation test?
[Tester] tested with the Chrome browser.
[Tester] tested with the Safari browser.

### Keyboard navigation checks

- [ ] Each link, button, form input, checkbox, radio button, select menu, and custom element can receive keyboard focus
- [ ] Each link, button, form input, checkbox, radio button, select menu, and custom element responds to expected keys
- [ ] All elements under focus have a visible focus indicator
- [ ] The order of [Tab] stops made sense and was appropriate for completing tasks

### Do you have any other results to share?

**Comment by MDomngz** [comment]
@darbyemadewell [this is the ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/106422) I was talking about - it mentions testing the two most common devices with the screen reader. The folks listed on the ADE ticket are a great source for additional guidance, if we need it. I do not mind helping get you connected. LMK if you need any support. Thanks!

**Comment by oddball-lindsay** [comment]
Closing this ticket out because it's too broad, instead we'll work through the smaller sized tickets [listed above](https://github.com/department-of-veterans-affairs/va.gov-team/issues/114786#issuecomment-3555087899).
